### PR TITLE
[Add]: blog: documentation

### PR DESCRIPTION
**Pull request by Donapi**
taskid: [3492925 ](https://www.odoo.com/web#id=3492925&cids=1&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/websites/blog.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- :guilabel:`Name`: your tag’s name;
+- :guilabel:`Category`: the category your tag belongs to. You can select an existing one, create a
+  new one, or leave it empty;
+- :guilabel:`Used in`: to apply your tag to existing blog posts, click :guilabel:`Add a line`,
+  select your posts, and click :guilabel:`Select`. Click :guilabel:`New` if your tag applies to a
+  post you want to create on the fly.
+
+.. note::
+   To show tags on your Blog dashboard page, click :menuselection:`Edit --> Customize`, and toggle
+   the switch of :guilabel:`Sidebar` and then of :guilabel:`Tags List`.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:52]: Perhaps add a screenshot to illustrate this? I found it, but it took me a while.

Maybe you could make it clearer by explaining more in detail how tags are displayed on the blog page, maybe even adding a little structure:
Tag visibility:
- Blog posts: tags are visible by default at the end of the blog post content.
- The main page of the blog: to make the tags visible, click :menuselection:`Edit --> Customize`, and toggle
   the switch of :guilabel:`Sidebar` and then of :guilabel:`Tags List`.

Or in one text block, "The tags are visible by default to the visitors at the end of each blog post. If you want to make the tags visible on the main page of the blog, click :menuselection:`Edit --> Customize`, and toggle the switch of :guilabel:`Sidebar` and then of :guilabel:`Tags List`.:"

And I'm not sure about "Blog dashboard". I would just say the main page, or maybe frontend (but then explain if we mean main page or the individual blog post), as dashboard - to me - implies something in the backend.

File: content/applications/websites/blog.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Customizing content
+===================
+
+You can enable the features related to your blog dashboard or a specific blog post according to your
+needs.
+
+Customizing your blog dashboard
+-------------------------------
+
+From your Blog dashboard page, click :menuselection:`Edit --> Customize`, and toggle the switch of
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:63]: Same comment as above about the dashboard, I'd suggest "The main page"

File: content/applications/websites/blog.rst [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+the following options to activate them:
+
+:guilabel:`Top Banner`: select :guilabel:`Name/Latest Post` to display the name of the latest post
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:66]: I know the option is called "Name/Latest Post" but I'd suggest to clarify in the description "to display the TITLE of the latest post", "name" is just weird. But it's nitpicking

File: content/applications/websites/blog.rst [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+on the top banner, or :guilabel:`Drop Zone for Building Blocks` to hide the top banner.
+
+:guilabel:`Layout`: select :guilabel:`Grid` or :guilabel:`List`.
+
+- :guilabel:`Cards`: to display your posts as cards;
+- :guilabel:`Increase Readability`: to improve the readability of the posts that are displayed.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:72]: Maybe explain a bit about how it works exactly (the font/size will be adjusted?)

File: content/applications/websites/blog.rst [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+:guilabel:`Sidebar`: to display the :guilabel:`Sidebar` and additional options:
+
+- :guilabel:`Archive`: a search bar allows to search posts by creation date;
+- :guilabel:`Author`: to mention the author on the post;
+- :guilabel:`Blog List`: to show the list of your blogs;
+- :guilabel:`Share Links`: to allow users to share your post on social media;
+- :guilabel:`Tags`: to display the tags of the post.
+
+:guilabel:`Breadcrumb`: to show the path to the post and the user's location.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:106]: Perhaps: and the user's location within the website.
Just so it's clear what kind of location we mean ;)

File: content/applications/websites/blog.rst [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+:guilabel:`Bottom`: click :guilabel:`Next Article` to display the following article below your post,
+or click :guilabel:`Comments` to allow visitors to comment on your post.
+
+:guilabel:`Select To Tweet`: to enable the tweet of part of the text when it is selected.
+
+.. seealso::
+   :doc:`Website Documentation <website>`.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:114]: Not sure if the . is needed at the end?

**Review by zulu-odoo — CHANGES_REQUESTED** [review]
Helloo,
Great page, I like it a lot! I left some comments but as always, feel free to ignore anything you don't agree with :) It's just suggestions.
Thank youuuu

File: content/applications/websites/blog.rst (hunk 6 above)
Comment (Donapi) [reply content/applications/websites/blog.rst:114]: I removed the dot and moved this see also at the beg of the page

File: content/applications/websites/blog.rst [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+Creating a blog
+===============
+
+To create or edit a blog, go to :menuselection:`Website --> Configuration --> Blogs: Blogs`. Click
+:guilabel:`New` and fill in the required information: add a :guilabel:`Blog Name`, a :guilabel:`Blog
+Subtitle`, and if needed, select the :guilabel:`Website` you want to restrict the blog to. If this
+field is left empty, your blog is visible from all websites.
+
+To access your blogs, go to your :guilabel:`website` and click on :guilabel:`Blog` on the top menu.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:19]:
```suggestion
To access your blogs, go to your :guilabel:`website` and click :guilabel:`Blog` on the top menu.
```

File: content/applications/websites/blog.rst [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+===================
+
+Go to your website and click :guilabel:`+New` in the top-right corner. Select :guilabel:`Blog Post`,
+:guilabel:`Select blog`, and add a blog post :guilabel:`Title`.
+
+Write your blog post and customize your page’s content using the website builder.
+
+.. tip::
+   - Illustrate your articles with copyright-free images from :doc:`Unsplash <website/configuration/unsplash>`.
+   - Type `/` in the text editor to format and adding elements to your text.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:32]:
```suggestion
   - Type `/` in the text editor to format and add elements to your text.
```

File: content/applications/websites/blog.rst [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+.. important::
+   Toggle the switch in the upper-right corner from :guilabel:`Unpublished` to :guilabel:`Published`
+   to make your article visible.
+
+Using tags
+==========
+
+Tags are visible by default at the end of each blog post, and can also be shown on the blog main
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:41]:
```suggestion
Tags are visible by default at the end of each blog post, and can also be shown on the blog's main
```

File: content/applications/websites/blog.rst [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+
+- :guilabel:`Name`: your tag’s name;
+- :guilabel:`Category`: the category your tag belongs to. You can select an existing one, create a
+  new one, or leave it empty;
+- :guilabel:`Used in`: to apply your tag to existing blog posts, click :guilabel:`Add a line`,
+  select your posts, and click :guilabel:`Select`. Click :guilabel:`New` if your tag applies to a
+  post you want to create on the fly.
+
+To show tags on your Blog main page, click :menuselection:`Edit --> Customize`, and toggle the
+switch of :guilabel:`Sidebar` and then of :guilabel:`Tags List`.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:56]:
```suggestion
:guilabel:`Sidebar` switch and then guilabel:`Tags List` switch.
```

I'm not 100% sure what is better, but up to you :)

File: content/applications/websites/blog.rst [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+
+Customizing content
+===================
+
+You can enable the features related to your blog main page or a specific blog post according to your
+needs.
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:62]: To me, this whole sentence is not necessary (as the section is called "customizing content"), but up to you :)

File: content/applications/websites/blog.rst [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+on the top banner, or :guilabel:`Drop Zone for Building Blocks` to hide the top banner.
+
+:guilabel:`Layout`: select to display blog posts on the main page as :guilabel:`Grid` or
+:guilabel:`List`.
+
+- :guilabel:`Cards`: to display your posts as cards;
+- :guilabel:`Increase Readability`: to improve the readability of the posts that are displayed.
+
+:guilabel:`Sidebar`: when enabled, a column appears on your page, displaying an :guilabel:`About us`
+section and, if enabled the below information:
```
Comment (zulu-odoo) [inline content/applications/websites/blog.rst:80]:
```suggestion
section and, if enabled, the below information:
```

**Review by zulu-odoo — COMMENTED** [review]
Hi Dona,
I hope that you can forgive me for taking 2 whole months to do this <3
Some small comments, and we're good!

File: content/applications/websites/blog.rst [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+:guilabel:`Layout`: select :guilabel:`Title Inside Cover` to display the title inside the cover
+image, or select :guilabel:`Title above Cover` to display the title above the cover image.
+
+- :guilabel:`Increase Readability`: to increase the size of your text and improve the readability.
+
+:guilabel:`Sidebar`: to display the :guilabel:`Sidebar` and additional options:
+
+- :guilabel:`Archive`: a search bar allows to search posts by creation date;
+- :guilabel:`Author`: to mention the author on the post;
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:108]:
```suggestion
- :guilabel:`Author`: to display the author on the post;
```

File: content/applications/websites/blog.rst [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+- :guilabel:`Blog List`: to show the list of your blogs;
+- :guilabel:`Share Links`: to allow users to share your post on social media;
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:110]:
```suggestion
- :guilabel:`Share Links`: to allow users to share your posts on social media;
```

File: content/applications/websites/blog.rst [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+- :guilabel:`Tags`: to display the tags of the post.
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:111]:
```suggestion
- :guilabel:`Tags`: to display the tags of your posts.
```

File: content/applications/websites/blog.rst [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+From your Blog main page, click :menuselection:`Edit --> Customize`, and toggle the switch of
+the following options to activate them:
+
+:guilabel:`Top Banner`: select :guilabel:`Name/Latest Post` to display the title of the latest post
+on the top banner, or :guilabel:`Drop Zone for Building Blocks` to hide the top banner.
+
+:guilabel:`Layout`: select to display blog posts on the main page as :guilabel:`Grid` or
+:guilabel:`List`.
+
+- :guilabel:`Cards`: to display your posts as cards;
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:76]:
```suggestion
- :guilabel:`Cards`: to display posts as cards;
```

File: content/applications/websites/blog.rst [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+- :guilabel:`Increase Readability`: to improve the readability of the posts that are displayed.
+
+:guilabel:`Sidebar`: when enabled, a column appears on your page, displaying an :guilabel:`About us`
+section and, if enabled, the below information:
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:80]:
```suggestion
:guilabel:`Sidebar`: to display a column containing an :guilabel:`About us`
section and, if enabled, additional sections:
```

File: content/applications/websites/blog.rst [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+- :guilabel:`Follow Us`: invites users to follow you on social media;
+- :guilabel:`Tags List`: shows all existing tags related to your posts. Users can click on a tag
+  to access all related posts.
+
+:guilabel:`Posts List`: select :guilabel:`Cover` to show the image related to your post, or
+:guilabel:`No Cover` to hide it.
+
+- :guilabel:`Author`: to show the author of the post;
+- :guilabel:`Comments/View Stats`: to display the number of comments and views on your post;
+- :guilabel:`Teaser & Tags`: to display a teaser text and tags on your posts.
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:92]:
```suggestion
- :guilabel:`Teaser & Tags`: to display a teaser text and tags of your posts.
```
Unclear what the teaser is.

File: content/applications/websites/blog.rst [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+
+:guilabel:`Breadcrumb`: to show the path to the post and the user's location within the website.
+
+:guilabel:`Bottom`: click :guilabel:`Next Article` to display the article below your post, or click
+:guilabel:`Comments` to allow visitors to comment on your post.
+
+:guilabel:`Select To Tweet`: to enable the tweet of part of the text when it is selected.
+
+.. note::
+   Changes applied to a blog post apply to all posts.
```
Comment (xpl-odoo) [inline content/applications/websites/blog.rst:121]: This should be mentioned in the section intro.

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Hello Dona! Nice job :)
A couple of general suggestions:
Screenshots could illustrate some options nicely. Also, be consistent when saying "the/a/your" and "post/s" whenever possible.

File: content/applications/websites/blog.rst [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+- :guilabel:`Author`: to show the author of the post;
+- :guilabel:`Comments/View Stats`: to display the number of comments and views on your post;
+- :guilabel:`Teaser & Tags`: to display a teaser text and tags on your posts.
+
+Customizing your blog post
+--------------------------
+
+Click :menuselection:`Edit --> Customize` and toggle the switch of the following options to activate
+them:
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/blog.rst:98]: already mentioned in a note below :)

File: content/applications/websites/blog.rst [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+
+To create a tag, go to :menuselection:`Website --> Configuration --> Blogs: Tags` and click
+:guilabel:`New`. You can also open an existing tag to edit it. Fill in the required information:
+
+- :guilabel:`Name`: your tag’s name;
+- :guilabel:`Category`: the category your tag belongs to. You can select an existing one, create a
+  new one, or leave it empty;
+- :guilabel:`Used in`: to apply your tag to existing blog posts, click :guilabel:`Add a line`,
+  select your posts, and click :guilabel:`Select`. Click :guilabel:`New` if your tag applies to a
+  post you want to create on the fly.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/blog.rst:53]: It's also possible to create tags frontend. There is currently a bug. I informed the PO, this is being fixed. I update the text as it should be.

File: content/applications/websites/blog.rst [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+Tags are visible by default at the end of each blog post, and can also be shown on the blog's main
+page to help you categorize and organize your blog posts and offer a user-friendly way to facilitate
+search.
+
+To create a tag, go to :menuselection:`Website --> Configuration --> Blogs: Tags` and click
+:guilabel:`New`. You can also open an existing tag to edit it. Fill in the required information:
+
+- :guilabel:`Name`: your tag’s name;
+- :guilabel:`Category`: the category your tag belongs to. You can select an existing one, create a
+  new one, or leave it empty;
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/blog.rst:50]: being clarified with PO

**Comment by xpl-odoo** [comment]
@Donapi let me know if this is ready for the re-review ;)

**Comment by xpl-odoo** [comment]
I pushed some changes to go faster @Donapi :)
@robodoo r+

**Pull request closed without merging**